src/quicreach.cpp

```diff
@@ -25,8 +27,6 @@
 #define QUIC_CALL
 #endif
 
-using namespace std;
```

Comment:
I personally prefer the `using namespace` model, as it makes all the code below less verbose. Why remove it?